### PR TITLE
CI/CD: Correct `postgres` mount point on Uffizzi Virtual Clusters

### PR DESCRIPTION
**Pull request by axisofentropy**
We noticed that databases were not persisting when `postgres` containers restarted on Uffizzi Virtual Clusters. I regret this was because we were mounting the persistent disk at `/var/lib/postgresql` while the `postgres` image was also mounting a temporary volume at `/var/lib/postgresql/data`, so the persistent volume was never used.

This PR corrects this by instead mounting the persistent volume at `/data` and setting the `PGDATA` environment variable so data is written to the `/data/pgdata` directory within. I've tested this change successfully on my own fork.

Additionally I've changed the temporary image name generator to use a string prefix and time-base (version 1) UUID strings. This will help us identify and garbage-collect images within our registry.

Thank you for your consideration and please reach out if you've any questions or concerns,
Adam Vollrath
Uffizzi

### CODE REVIEW

File: .github/uffizzi/k8s/manifests/pg-deploy.yaml [hunk 1]
```diff
@@ -19,11 +19,14 @@ spec:
           imagePullPolicy: 'IfNotPresent'
           ports:
             - containerPort: 5432
+          env:
+            - name: PGDATA
+              value: "/data/pgdata"
```
Comment (freben) [inline .github/uffizzi/k8s/manifests/pg-deploy.yaml:24]: I see prettier is complaining about this - it wants single quotes.

**Review by freben — COMMENTED** [review]
Alright, just a small style error.

File: .github/uffizzi/k8s/manifests/pg-deploy.yaml (hunk 1 above)
Comment (axisofentropy) [reply .github/uffizzi/k8s/manifests/pg-deploy.yaml]: corrected, thank you for the feedback!

**Comment by axisofentropy** [comment]
I've also added explicit resource requests to more efficiently schedule these containers.

**Review by freben — APPROVED** [review]

**Pull request merged**